tests/scaling-pipelines/scenario/git-resolver/pipeline.yaml.j2

```diff
@@ -0,0 +1,25 @@
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: git-resolver
+spec:
+  params: []
+  tasks: 
+  {% if resolver_type == "git" %}
+  - name: "git-task-reference"
+    taskRef:
+      resolver: git
+      params:
+      - name: url
+        value: https://github.com/tektoncd/catalog.git
+      - name: revision
+        value: main
+      - name: pathInRepo
+        value: task/git-clone/0.6/git-clone.yaml
```

Comment:
The task linked here seems to be doing another git-clone operation to load a git repo files locally on to a working directory. If we are to compare with the resolver_type "task" as given below, I feel we should be using a simple hello-world Task instead of git-clone (Basically put the hello-world Task onto some git repository, and refer that in the git resolver). 

Other option is to use https://github.com/tektoncd/catalog/blob/main/task/git-clone/0.6/git-clone.yaml in the below task.yaml. In that way, we have similar comparison for the task itself.